gen-app-builder/search-web-app/templates/recommend.html

```diff
@@ -0,0 +1,290 @@
+                      {% set resultUrl = result["link"] %}
+                      <a href="{{resultUrl}}" target="_blank">
+                        <h2 class="mdc-typography mdc-typography--headline6">
+                          {{result["title"]|safe}}
+                        </h2>
+                      </a>
+                      <h3 class="mdc-typography mdc-typography--subtitle2">
+                        Document ID: <code>{{result["id"]|safe}}</code>
+                      </h3>
+                      <h3 class="mdc-typography mdc-typography--subtitle2">
+                        MIME Type: <code>{{result["mimeType"]|safe}}</code>
+                      </h3>
+                    </div>
+                  </div>
+                </div>
+              </div>
+              <div class="mdc-layout-grid__cell mdc-layout-grid__cell--span-6">
+                <div class="mdc-card mdc-card--outlined">
+                  <pre><code class="language-json" lang="json">{{result["resultJson"]}}</code></pre>
+                </div>
+              </div>
+            </div>
+          </div>
+          {% endfor %}
+        </div>
+        <div id="json-tab" class="tab-hidden">
+          {% if request_url %}
+          <div class="mdc-layout-grid">
+            <div class="mdc-layout-grid__inner">
+              <div
+                class="mdc-layout-grid__cell mdc-layout-grid__cell--span-2"
+              ></div>
+              <div class="mdc-layout-grid__cell mdc-layout-grid__cell--span-8">
+                <b>Request URL:</b>
+                <pre><code class="language-shell" lang="shell">{{request_url}}</code></pre>
+              </div>
+              <div
+                class="mdc-layout-grid__cell mdc-layout-grid__cell--span-2"
+              ></div>
+            </div>
+          </div>
+          {% endif %}
+          <div class="mdc-layout-grid">
+            <div class="mdc-layout-grid__inner">
+              {% if raw_request %}
+              <div class="mdc-layout-grid__cell mdc-layout-grid__cell--span-6">
+                <b>Request Body:</b>
+                <pre><code class="language-json" lang="json">{{raw_request}}</code></pre>
+              </div>
+              {% endif %} {% if raw_response %}
+              <div class="mdc-layout-grid__cell mdc-layout-grid__cell--span-6">
+                <b>Response Body:</b>
+                <pre><code class="language-json" lang="json">{{raw_response}}</code></pre>
+              </div>
+              {% endif %}
+            </div>
+          </div>
+        </div>
+      </div>
+      {% endif %}
+    </main>
+    <section class="footer">
+      <p>
+        Made by <a href="https://github.com/holtskinner">holtskinner@</a> -
+        Source on
+        <a
+          href="https://github.com/GoogleCloudPlatform/generative-ai/tree/main/gen-app-builder/search-web-app"
+          >GitHub</a
+        >
+      </p>
+    </section>
+  </body>
```

Comment:
(Optional) For folks that are only using the app and not looking at this repo/README, maybe add a callout for `Powered by Google Cloud` or similar with a link to Gen App Builder or the `Google Cloud Products Used` that are listed in the README.

Reply:
I've considered adding that, also been getting lots of questions about the datasets used, etc. I'll go ahead and add it.